mlflow/store/artifact/unity_catalog_models_artifact_repo.py

```diff
@@ -81,7 +81,7 @@ def _get_scoped_token(self):
         req_body = {
             "name": self.model_name,
             "version": self.model_version,
-            "operation": MODEL_VERSION_READ_WRITE,
```

Comment:
This was a mistake in an earlier PR, we don't need write credentials to download models from UC (read-only credentials should suffice)